debian/changelog

```diff
@@ -1,3 +1,128 @@
+snapcraft (2.35) xenial; urgency=medium
+
+  [ Sergio Schvezov ]
+  * pluginhandler: error out on scriptlet errors
+  * meta: ensure main keys are ordered in snap.yaml
+  * demos: remove the py?-project demos
+  * store: switch to new endpoints
+  * static: fix flake8 errors in setup.py
+  * docker: add the environment variable to setup core (#1576)
+  * cli: add the pack command (#1565)
+  * store: handle revoked developers (#1554)
+  * lifecycle: split into its own package (#1626)
+  * libraries: exclude the full set of libc6 (#1632)
+  * lxd: better surfacing of errors (#1647)
+  * New upstream release (LP: #1729417)
+
+  [ Kyle Fazzari ]
+  * project_loader: aliases are deprecated
+  * catkin plugin: don't assume catkin is in underlay
+  * catkin plugin: only append PYTHONPATH if set
+  * plugins: extract python finder functions
+  * plugins: extract sitecustomize logic from python
+  * dirs: set plugin, schema, and library dir for snap
+  * plugins: extract pip from python plugin
+  * catkin plugin: support rosdep pip dependencies (#1581)
+  * plugins: add ros2 boostrapper (#1582)
+  * travis: run snapd tests only if not cron (#1592)
+  * snapcraft.yaml: don't re-use build dir (#1601)
+  * schema: improve invalid app, hook, and part errors (#1615)
+  * plugins: build-attributes is already in the state (#1620)
+  * tests: skip catkin test on non-xenial (#1621)
+  * tests: don't hit internet in ros2 units (#1619)
+  * states: add scriptlets to build state (#1618)
+  * schema: sync patterns with snapd (#1622)
+  * snap: remove leaking LD_LIBRARY_PATH (#1635)
+  * store: guide to account creation upon login (#1616)
+  * repo: add elementary to deb distros (#1637)
+
+  [ Leo Arias ]
+  * typo: replace occured with occurred
+  * python node: record installed node packages in manifest
```

Comment:
here's an error, should be node plugin instead of python node

Reply:
fixed